libraries/opensk/fuzz/Cargo.lock

```diff
@@ -96,34 +96,6 @@ dependencies = [
  "untrusted",
 ]
 
-[[package]]
```

Comment:
Why do we keep this `Cargo.lock` since it's a library?

Reply:
Removed all `Cargo.lock` files in `*/fuzz/` .